src/main/java/com/arpnetworking/clusteraggregator/Main.java

```diff
@@ -98,20 +123,10 @@ public static void main(final String[] args) {
                 .addTrigger(new FileTrigger.Builder().setFile(configurationFile).build())
                 .addListener(configurator)
                 .build();
+        configRef.set(configuration);
```

Comment:
Technically these should be set above to avoid a gap between creation and exception handling cleanup.

e.g.
```java
configuratorRef.set(new Configurator<>(Main::new, ClusterAggregatorConfiguration.class));
configRef.set(new DynamicConfiguration.Builder()
  .setObjectMapper(objectMapper)
  .addSourceBuilder(
    new JsonNodeFileSource.Builder().setObjectMapper(objectMapper)
      .setFile(configurationFile))
  .addTrigger(new FileTrigger.Builder().setFile(configurationFile).build())
  .addListener(configurator)
  .build());
```